lib/orm.php

```diff
@@ -2282,6 +2335,33 @@ protected function build_insert() {
 		return \implode( ' ', $query );
 	}
 
+	/**
+	 * Builds a bulk INSERT query.
+	 *
+	 * @param array $models Array of model instances to be inserted.
+	 *
+	 * @return string The insert query.
+	 */
+	protected function build_insert_many( $models ) {
+		$example_model      = $models[0];
+		$total_placeholders = '';
+
+		$query        = [];
+		$query[]      = 'INSERT INTO';
+		$query[]      = $this->quote_identifier( $example_model->orm->table_name );
+		$field_list   = \array_map( [ $this, 'quote_identifier' ], \array_keys( $example_model->orm->dirty_fields ) );
```

Comment:
This should not just be the dirty columns of the first model but those of all models.

Reply:
Resolved in https://github.com/Yoast/wordpress-seo/pull/17364/